recipes/libxslt/all/conanfile.py

```diff
@@ -0,0 +1,177 @@
+
+    def _patch_sources(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+    def build(self):
+        self._patch_sources()
+        if self._is_msvc:
+            self._build_windows()
+        else:
+            self._build_with_configure()
+
+    def _build_windows(self):
+        with tools.chdir(os.path.join(self._full_source_subfolder, 'win32')):
+            debug = "yes" if self.settings.build_type == "Debug" else "no"
+            static = "no" if self.options.shared else "yes"
+
+            with tools.vcvars(self.settings):
+                args = ["cscript",
+                        "configure.js",
+                        "compiler=msvc",
+                        "prefix=%s" % self.package_folder,
+                        "cruntime=/%s" % self.settings.compiler.runtime,
+                        "debug=%s" % debug,
+                        "static=%s" % static,
+                        'include="%s"' % ";".join(self.deps_cpp_info.include_paths),
+                        'lib="%s"' % ";".join(self.deps_cpp_info.lib_paths),
+                        'iconv=no',
+                        'xslt_debug=no',
+                        'debugger=no',
+                        'crypto=no']
+                configure_command = ' '.join(args)
+                self.output.info(configure_command)
+                self.run(configure_command)
+
+                # Fix library names because they can be not just zlib.lib
+                def format_libs(package):
+                    libs = []
+                    for lib in self.deps_cpp_info[package].libs:
+                        libname = lib
+                        if not libname.endswith('.lib'):
+                            libname += '.lib'
+                        libs.append(libname)
+                    return ' '.join(libs)
+
+                def fix_library(option, package, old_libname):
+                    if option:
+                        tools.replace_in_file("Makefile.msvc",
+                                              "LIBS = %s" % old_libname,
+                                              "LIBS = %s" % format_libs(package))
+
+                if "icu" in self.deps_cpp_info.deps:
+                    fix_library(True, 'icu', 'wsock32.lib')
+
+                tools.replace_in_file("Makefile.msvc", "libxml2.lib", format_libs("libxml2"))
+                tools.replace_in_file("Makefile.msvc", "libxml2_a.lib", format_libs("libxml2"))
+
+                with tools.environment_append(VisualStudioBuildEnvironment(self).vars):
+                    self.run("nmake /f Makefile.msvc install")
+
+    def _build_with_configure(self):
+        in_win = self.settings.os == "Windows"
+        env_build = AutoToolsBuildEnvironment(self, win_bash=in_win)
+        if not in_win:
+            env_build.fpic = self.options.fPIC
+        full_install_subfolder = tools.unix_path(self.package_folder) if in_win else self.package_folder
```

Comment:
This should be fixed in conan 1.25.0
```suggestion
        full_install_subfolder = tools.unix_path(self.package_folder)
```

Reply:
ok, I'll commit this when 1.25.0 is out

Reply:
It won't build now because I removed `in_win` in another comment.